scripts/setup/interactive_env_setup.py

```
@@ -0,0 +1,409 @@
+    """Print section header"""
+    print(f"\n{Colors.OKBLUE}{Colors.BOLD}{text}{Colors.ENDC}")
+    print(f"{Colors.OKBLUE}{'-' * len(text)}{Colors.ENDC}")
+
+
+def generate_secret(length: int = 32) -> str:
+    """Generate a cryptographically secure random secret"""
+    return secrets.token_hex(length)
+
+
+def prompt_input(
+    prompt: str, default: str = "", required: bool = False, secret: bool = False
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)